cosmic_ray/importing.py

```diff
@@ -61,8 +61,11 @@ def using_mutant(module_name, mutant):
     mutated AST.
 
     """
-    if module_name in sys.modules:
-        del sys.modules[module_name]
+    module_to_delete = ""
+    for parent in module_name.split('.'):
+        module_to_delete = ("%s.%s" % (module_to_delete, parent)).lstrip('.')
+        if module_to_delete in sys.modules:
+            del sys.modules[module_to_delete]
```

Comment:
For some reason the assignment to sys.modules doesn't have the desired effect. Looping through del_mods and deleting them one by one however seems to be fine.